index.js

```diff
@@ -40,19 +39,21 @@ function checkGuess() {
   }
 
   if (guess !== targetNumber) {
-    if (guess < targetNumber) {
-      tooLowMessage.style.display = '';
-    } else {
+    if (guess < 1) {
+      greaterThanZeroMessage.style.display = '';
```

Comment:
This code does work for notifying of an invalid guess, but it also still counts against your guesses remaining. Maybe you could add a guess back for this case? Or skip the code that removes one?